src/components/search/search-result.component.tsx

```diff
@@ -101,33 +101,44 @@ const SearchResultComponent: React.FC<SearchResultProps> = ({ isAnonymous, user,
     let disableRequest = false;
 
     if (status) {
-      disableRequest = [FriendStatus.PENDING, FriendStatus.APPROVED].includes(status);
+      disableRequest = [FriendStatus.PENDING, FriendStatus.APPROVED, FriendStatus.REJECTED].includes(status);
```

Comment:
are you disabling or cancelling the request? disabling a request sounds strange...

Reply:
itu yg gw liat sih buat disable Button add friend.

Reply:
this is when a user has already sent a friend request to another user. We check it, and if there's a status, we disable the button.